docs/security/trust/content_trust.md

```diff
@@ -109,17 +109,20 @@ $ docker pull someimage@sha256:d149ab53f8718e987c3a3024bb8aa0e2caadf6c0328f1d9d8
 
 Trust for an image tag is managed through the use of signing keys. A key set is
 created when an operation using content trust is first invoked. Docker's content
-trust makes use of four different keys:
+trust makes use of five different types of keys:
 
 | Key                 | Description                                                                                                                                                                                                                                                                                                                                                                         |
 |---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| root key         | Root of content trust for a image tag. When content trust is enabled, you create the root key once. |
-| target and snapshot | These two keys are known together as the "repository" key. When content trust is enabled, you create this key when you add a new image repository. If you have the root key, you can export the repository key and allow other publishers to sign the image tags.    |
-| timestamp           | This key applies to a repository. It allows Docker repositories to have freshness security guarantees without requiring periodic content refreshes on the client's side.                                                                                                              |
-
-With the exception of the timestamp, all the keys are generated and stored locally
-client-side. The timestamp is safely generated and stored in a signing server that
-is deployed alongside the Docker registry. All keys are generated in a backend
+| root key            | Root of content trust for a image tag. When content trust is enabled, you create the root key once. |
+| target and snapshot | These two keys are known together as the "repository" key. When content trust is enabled, you create this key when you add a new image repository. |
+| timestamp           | This key applies to a repository. It allows Docker repositories to have freshness security guarantees without requiring periodic content refreshes on the client's side. |
+| delegation          | Delegation keys are optional and allow you to delegate signing image tags to other publishers without having to share your repository key |
+
+With the exception of the timestamp, snapshot, and delegation keys, all the keys
```

Comment:
Perhaps we should invert this now; if I understand correctly, only "root" and "target" keys are stored locally?